build/push-federation-images.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pushes federation container images to existing repositories
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source "${KUBE_ROOT}/build/util.sh"
+
+source "${KUBE_ROOT}/federation/cluster/common.sh"
+
+FEDERATION_IMAGE_TAG="$(kube::release::semantic_image_tag_version)" push-federated-images
```

Comment:
sorry for being picky, but while you are at it, will be great if you can rename "push-federated-images" to push-federation-images" :)